packages/server/src/arpa_reporter/lib/audit-report.js

```diff
@@ -100,6 +101,58 @@ async function getProjectSummaryRow(data) {
     };
 }
 
+async function getColumnForProjectSummaryGroupedByProjectRow(projectId, records, reportingPeriods) {
+    const record = records[0];
+    let reportingPeriodStartDate = '';
```

Comment:
i don't think you need this here. Each `reportingPeriodStartDate` is in its own function, so you should be able to make it a `const` in each case.

Reply:
Can you re-assign a const variable? The variable is a primitive so it's not like we're re-assigning a property on a const, which would work.